env/WCOSS2.env

```
@@ -76,6 +76,13 @@ elif [[ "${step}" = "aeroanlrun" ]]; then
     [[ ${NTHREADS_AEROANL} -gt ${nth_max} ]] && export NTHREADS_AEROANL=${nth_max}
     export APRUN_AEROANL="${launcher} -n ${npe_aeroanlrun}"
 
+elif [[ "${step}" = "prepobsaero" ]]; then
+
+    nth_max=$((npe_node_max / npe_node_prepaeroobs))
+
+    export NTHREADS_PREPOBSAERO=${nth_prepobsaero:-1}
+    export APRUN_PREPOBSAERO="${launcher} -n ${npe_prepobsaero} --cpus-per-task=${NTHREADS_PREPOBSAERO}"
```

I think this should be `--depth` for mpiexec:
```suggestion
    export APRUN_PREPOBSAERO="${launcher} -n ${npe_prepobsaero} --ppn ${npe_node_prepobsaero}--cpu-bind depth --depth=${NTHREADS_PREPOBSAERO}"
```